app/controllers/api/services_controller.rb

```diff
@@ -16,6 +17,12 @@ class Api::ServicesController < Api::BaseController
     actions.sublayout 'api/service'
   end
 
+  def index
+    activate_menu :dashboard
+    @services = current_user.accessible_services
```

Comment:
plz add a test in `test/integration/api/services_controller_test.rb`

Reply:
Done 👍